constraint_utils/constraint_basics.py

```diff
@@ -191,12 +191,51 @@ def split_context_mt(raw_context_ht_path: str, coverage_ht_paths: Dict[str, str]
 
 def pre_process_data(ht: hl.Table, split_context_ht_path: str,
                      output_ht_path: str, overwrite: bool = False) -> None:
+    """ 
+    Add following annotations from VEP context Table onto gnomAD data:
+        - context
+        - methylation
+        - coverage
+        - gerp
+        - pass_filters
+    
```

Comment:
```suggestion
    """ 
    Add annotations from VEP context Table to gnomAD data.

    Function adds the following annotations:
        - context
        - methylation
        - coverage
        - gerp
        - pass_filters
    
```
the first line of docstrings should be a command